From 39be9f2d8e3d9369bb9a35da025e2fcec314418c Mon Sep 17 00:00:00 2001
From: George Calin <george.calin@gmail.com>
Date: Wed, 11 Jun 2025 16:24:07 +0300
Subject: [PATCH] modified C source file

---
 factorial.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/factorial.c b/factorial.c
index 8b276eb..9b9a2c7 100644
--- a/factorial.c
+++ b/factorial.c
@@ -19,7 +19,7 @@ int usage(void) {
 
 int main(int argc, char *argv[]) {  
     if(argc<=1) {
-        printf("ERROR: No argument found...\n");
+        printf("INVALID INPUT: No argument found...\n");
         usage();
         exit(1);
      }
